compiler/rustc_typeck/src/check/check.rs

```diff
@@ -1444,10 +1444,6 @@ pub(super) fn check_type_params_are_used<'tcx>(
     }
 }
 
-pub(super) fn check_mod_item_types(tcx: TyCtxt<'_>, module_def_id: LocalDefId) {
-    tcx.hir().visit_item_likes_in_module(module_def_id, &mut CheckItemTypesVisitor { tcx });
```

Comment:
Why remove the query instead of implementing in terms of `tcx.hir_module_items(module_def_id)`?